src/components/PatientCreationPage/TriagePills.tsx

```diff
@@ -158,9 +158,9 @@ const TriagePills = ({
               root: classes[`${level.toLowerCase()}pill`],
               selected:
                 classes[
-                `selected${
-                level.charAt(0).toUpperCase() + level.slice(1).toLowerCase()
-                }Pill`
+                  `selected${
```

Comment:
i apologize if this causes a merge conflict (i would actually merge master and double check this doesn't - i think it should be merged in by now)

Reply:
I just merged master and there's no conflicts